### PR TITLE
Write IDP in full and remove ellipsis #32

### PR DESCRIPTION
**Pull request by svdwyer**
fix: copy text labels, updated e2e tests
fixes https://github.com/SolidLabResearch/solid-identity-manager/issues/32

### CODE REVIEW

File: src/popup/popup.html [hunk 1]
```diff
@@ -90,13 +90,13 @@ <h1>Add new profile</h1>
                 </fieldset>
 
                 <fieldset>
-                    <label for="idp">IDP</label>
+                    <label for="idp">Identity provider</label>
                     <input id="idp" name="idp" type="text" placeholder="Solid identity provider"/>
```
Comment (pheyvaer) [inline src/popup/popup.html:94]: This is not updated.

**Review by pheyvaer — CHANGES_REQUESTED** [review]

File: src/popup/popup.html (hunk 1 above)
Comment (svdwyer) [reply src/popup/popup.html:94]: Do you mean the placeholder? Do you want it to also be "Identity provider" instead of "Solid identity provider"?

Comment (pheyvaer) [reply src/popup/popup.html]: I said in my initial comment that I wanted it to be consistent with the placeholder for WebID.

Comment (svdwyer) [reply src/popup/popup.html]: ✔️

**Review by pheyvaer — APPROVED** [review]

**Pull request merged**